obs-browser-plugin.cpp

```diff
@@ -126,6 +126,27 @@ static void browser_source_get_defaults(obs_data_t *settings)
 	obs_data_set_default_bool(settings, "reroute_audio", false);
 }
 
+static void browser_source_get_defaults_v2(obs_data_t *settings)
```

Comment:
Could this technically call `browser_source_get_defaults()` first, then set the width/height in this function?

Reply:
I was also thinking this as code duplication is generally bad, but after realising our defaults work in the way they do, it does also seem somewhat fitting to keep them separated. If we're committed to never changing v1 defaults then I think removing the duplication is OK.

Reply:
Updated function to remove duplicate code.